http_service/app.py

```diff
@@ -132,6 +130,39 @@ def is_running(model_name, bug_id):
     return False
 
 
+def get_bug_ids_last_change_time(bug_ids):
+    query = {"id": bug_ids, "include_fields": ["last_change_time", "id"]}
+    bug = bugzilla.get(
```

Comment:
I wonder if it wouldn't even be faster to perform the query directly with requests instead of with libmozdata. libmozdata is first performing a query to get the number of results, then chunking in batches of 100 bugs, simply because often people retrieve a lot of data from bugs. In this case our query only requests the id and the last_change_time, so we might as well do one query to bugzilla instead of chunking.

Reply:
That might indeed be faster, will Bugzilla accept returning 1000 result at the same time? Do you have some pointers where we are querying bugzilla directly with requests?

Reply:
> That might indeed be faster, will Bugzilla accept returning 1000 result at the same time?

Yes, we're doing that in relman-auto-nag (with the chunk size parameter of get_bugs).

> Do you have some pointers where we are querying bugzilla directly with requests?

You can just do something like `requests.get('https://bugzilla.mozilla.org/rest/bug', params=query)`

Reply:
I went the requests way, thank you for the idea